monitoring/ceph-mixin/dashboards_out/host-details.json

```diff
@@ -783,7 +783,7 @@
          "lines": true,
          "linewidth": 1,
          "links": [ ],
-         "nullPointMode": "connected",
+         "nullPointMode": "null",
```

Comment:
Are we ok with [this change](https://grafana.com/docs/grafana/latest/visualizations/graph-panel/#:~:text=non%2Dnull%20value.-,null,-%2D%20(default)%20If%20there)? On one hand I prefer that charts don't beautify/fake the underlying data, but I understand that in some cases having gaps in the charts might be annoying... No strong objection.

Reply:
I think we put do that because looking at the dashboards I suspect this got carried from the original json and this is not consistent across the dashboards... This seems a pretty safe default, I would have no strong objection if someone switch this to something else later on as long as it's consistent or it's justified on the specific dashboards/panels.